pages/docs/learn/durable-execution.mdx

```diff
@@ -0,0 +1,34 @@
+export const description = 'Learn how Durable Execution works with Inngest. '
+
+# How Durable Execution works with Inngest
+
+One of the core features of Inngest is Durable Execution. Durable Execution allows your functions to be fault-tolerant and resilient to failures. The end result is that your code, and therefore, your overall application, is more reliable.
+
+In this guide, we'll cover the basics of Durable Execution, how it works, and how it works with Inngest functions.
```

Comment:
```suggestion
In this guide, we'll cover what Durable Execution is, how it works, and how it works with Inngest functions.
```

this would be less ego-triggering for some folks